threem.py

```diff
@@ -70,7 +71,10 @@ def get_events_between(self, start, end, cache_result=False):
         try:
             events = EventParser().process_all(response.content)
         except Exception, e:
-            print response.content
+            self.log.error(
```

Comment:
No, self.log is a property of Monitor (core/monitor.py) which looks up a logger based on the monitor's .service_name. If you're not inside a Monitor (or something similar like CoverageProvider) there is no self.log.